src/twilio/functions/after-input.js

```diff
@@ -0,0 +1,72 @@
+exports.handler = function recordRequest(context, event, callback) {
+  console.log(`CallSID:${event.CallSid}`);
+
+  const twilioSid = event.CallSid;
+  const phone = event.From;
+  const neighborhood = event.area;
+  const recordingUrl = event.RecordingUrl;
+  let status = "";
+  let manyc = "";
+  let carcluster = "";
+
+  const Airtable = require('airtable'); // eslint-disable-line
+  Airtable.configure({
+    endpointUrl: "https://api.airtable.com",
+    apiKey: context.AIRTABLE_API_KEY // eslint-disable-line
+  });
+
+  const base = Airtable.base("apppK7mrvMPcwtv6d");
```

Comment:
I think this airtable functionality (and the interactions below in this file) already exist in our library: https://github.com/crownheightsaid/mutual-aid-app/blob/master/src/lib/airtable/tables/requests.js

Reply:
that's true - ! these little phone functions in src/twilio are what we deploy separately to [Twilio Runtime](https://www.twilio.com/docs/runtime), so they are serverless - unless i'm missing something, we don't want to pull from the rest of app here, we want these to live on their own 

(but we list them here for transparency/version control etc.!)

Reply:
Oh I see! That makes sense, I didn't realize these were meant to be independent.